Dragonflight/DawnOfTheInfinite/BlightOfGalakrond.lua

```diff
@@ -219,6 +265,7 @@ end
 
 function mod:IncineratingBlightbreath(args)
 	-- can't use GetBossTarget, Dazhak targets a player ~1s before the cast and detargets them as the cast starts
+	incineratingBlightbreathCount = incineratingBlightbreathCount + 1
```

Comment:
Typically it's better to increment the count after the message and before the bar, that way if the count is ever used in the message/bar text in the future it will be the correct value.